conans/client/cmd/export.py

```diff
@@ -15,29 +15,25 @@
 from conans.model.scm import detect_repo_type
 from conans.paths import CONANFILE
 from conans.search.search import search_recipes, search_packages
-from conans.util.files import is_dirty, load, mkdir, rmdir, save, set_dirty, remove
+from conans.util.files import is_dirty, load, rmdir, save, set_dirty, remove
 from conans.util.log import logger
 
 
-def export_alias(reference, target_reference, cache, output):
-    if reference.name != target_reference.name:
-        raise ConanException("An alias can only be defined to a package with the same name")
+def export_alias(ref_layout, target_reference, output, revisions_enabled):
     conanfile = """
 from conans import ConanFile
 
 class AliasConanfile(ConanFile):
     alias = "%s"
-""" % target_reference.full_repr()
+""" % target_reference
```

Comment:
This is changing behavior, not allowing alias to revisions, why? Discuss with @lasote what behavior we want for alias.

Reply:
I relaunched the CI for this with revisions enabled, because yes, it should break the alias test with revisions.